packages/Path-View-Core.package/SClassDiagram.class/instance/dragBottomBy..st

```diff
@@ -0,0 +1,5 @@
+geometry
+dragBottomBy: aValue
+
+	self extent: self extent + (0 @ aValue)
```

Comment:
Maybe we should consider if we use `withBottom: y` it returns a new object every time.